hazelcast/src/test/java/com/hazelcast/concurrent/lock/ObjectNamespaceSerializationHelperTest.java

```diff
@@ -84,13 +79,19 @@ public void readNamespaceCompatibly() throws Exception {
     private ObjectNamespace serialiseDeserialise(ObjectNamespace ns, Version streamVersion) throws IOException {
         final byte[] bytes = serialize(ns, streamVersion);
         final BufferObjectDataInput input = ss.createObjectDataInput(bytes);
-        return ObjectNamespaceSerializationHelper.readNamespaceCompatibly(input);
```

Comment:
`ObjectNamespaceSerializationHelper` might still be useful. I see the same write/read logic used in several places. OR we can remove `DefaultObjectNamespace` completely.

Reply:
Left the `ObjectNamespaceSerializationHelper` class. Can you check now?